src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -35,11 +35,13 @@ import MutationMapperDataStore, {
     PROTEIN_IMPACT_TYPE_FILTER_ID,
 } from './MutationMapperDataStore';
 import WindowStore from '../window/WindowStore';
+import { Mutation } from 'cbioportal-utils';
```

Comment:
unused import?

Reply:
yes this was related to my failed attempts to make the typing more specific (see other response)